onyo/commands/init.py

```diff
@@ -21,12 +21,12 @@ def init(args: argparse.Namespace, opdir: str) -> None:
     Running ``onyo init`` on an existing repository is safe. It will not
     overwrite anything; it will exit with an error.
     """
-    target_dir = Path(opdir)
+    target_dir = Path.cwd()
     if args.directory:
         if Path(args.directory).is_absolute():
             target_dir = Path(args.directory)
         else:
-            target_dir = Path(opdir, args.directory)
+            target_dir = Path(args.directory)
```

Comment:
This does not make sense anymore.
1. `target_dir` gets the same assignment in both cases.
2. Now a relative one wouldn't be resolved at all

I'd replace the entire `target_dir` setting by:

```
target_dir = Path(args.directory).resolve() if args.directory else Path.cwd()
```